api/src/main/java/org/pmiops/workbench/compliance/ComplianceServiceImpl.java

```diff
@@ -84,4 +92,33 @@ public Integer getMoodleId(String email) throws ApiException {
     }
     return response.getBadges();
   }
+
+  @Override
+  public Map<String, BadgeDetails> getUserBadgesByName(String email) throws ApiException {
+    if (!enableMoodleCalls()) {
+      return null;
```

Comment:
Can you please return an empty map instead of null? Otherwise, the next best thing is `Optional<Map>`, which is no less awkward. In general, if I call a method returning a collection in Java, I don't expect nor do I check for `null`.